docs/source/api/chaco.api.rst

```diff
@@ -0,0 +1,13 @@
+..
+   (C) Copyright 2021 Enthought, Inc., Austin, TX
```

Comment:
nit. the year range needs to be fixed

Reply:
👍 
I saw 2006-2021 in the `LICENSE.txt` file so I assume 2006 is the correct start year.